GLM_2.Rmd

```diff
@@ -0,0 +1,108 @@
+---
+title: "GLM"
+output: pdf_document
+date: "2023-03-18"
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, warning = F, message = F)
+```
+
+
+# Data Exploration
```

Comment:
We'll combine this sectoin with the EDA section of the .Rmd file in the main branch.